src/ngx_pagespeed.cc

```diff
@@ -492,7 +492,8 @@ void copy_response_headers_from_ngx(const ngx_http_request_t* r,
 
   // When we don't have a date header, set one with the current time.
   if (headers->Lookup1(HttpAttributes::kDate) == NULL) {
-    headers->SetDate(ngx_current_msec);
```

Comment:
Never mind; I see this is just a big fat global bomb supplied by nginx that we are using.